plugins/commands/advisor/src/main/kotlin/AdvisorCommand.kt

```diff
@@ -62,7 +62,7 @@ import org.ossreviewtoolkit.utils.ort.ortConfigDirectory
 
 @OrtPlugin(
     id = "advise",
-    displayName = "advise command",
+    displayName = "advise",
```

Comment:
Maybe we should make these command names also uppercase.
And maybe we should just rename the classes to `AdviseCommand` and so on, then we don't have to set the ids manually. At least for most of the commands that would work.
What do you think?

Reply:
That turned out to be a bigger task than expected. For example, doing this consistently would mean to derive plugin ID "upload-result-to-postgres" from class name "UploadResultToPostgresCommand". That's doable, but would also imply getting "oss-index" for "OssIndex", which would be a breaking change. Is that acceptable?

Reply:
That's why I meant it would work for at least most commands. I haven't looked at the code but must the ID be "upload-result-to-postgres" because it is used directly as the command name for Clikt? Otherwise it could just be "UploadResultToPostgres". 
But I would not like to generally switch from pascal case to kebab case for default plugin IDs, this would be a massive breaking change, for example, when the "WebApp" report format is now called "web-app".

Reply:
What about `AOSD2.0` as the id? Should it stay, or should it be simplified to `Aosd20`? Same for version 2.1.

Reply:
I think in the past we wanted plugin IDs to be close to the original name if there is one, like "CycloneDX" instead of "CycloneDx" or "cyclone-dx". To me it makes sense to keep that approach and set the display name only if it adds value, so I would also keep "AOSD2.0".